changelog.md

```diff
@@ -2,6 +2,67 @@
 
 ## 0.13.0~rc1
 
+### Web Application
+
+* Added fingerprint of GPG reply key to Source object in Journalist API (#4436)
+* Updated message for flagged sources in Source Interface (#4428)
+* Added type hinting in Journalist Interface (#4404, #4407)
+* Added a /logout endpoint to the Journalist API (#4349)
+* Fixed 500 error caused by inconsistent session (#4391)
+* Added submission file sha256 hash as Etag on downloads in Journalist Interface (#4314)
+* Added Python3 compliance for Source and Journalist Interface (#4239)
+* Updated Python cryptography dependency, updated safety check (#4297)
+* Added support for HTTP DELETE method on Journalist Interface for use by API (#4023)
+* Removed cssmin dependency (#4227)
+* Removed jQuery dependency from Journalist and Source Interface (#4211)
+
+### Operations
```

Comment:
nit: traditionally we've used the operations section for ansible/instance provisioning from the perspective of an end-user. The contents of this section might be better suited to the developer section